analyzers/src/SonarAnalyzer.Common/Rules/UriShouldNotBeHardcodedBase.cs

```diff
@@ -129,22 +127,22 @@ private bool IsInCheckedContext(TExpressionSyntax expression, SemanticModel mode
             if (argument != null)
             {
                 var argumentIndex = GetArgumentIndex(argument);
-                if (argumentIndex == null ||
-                    argumentIndex < 0)
+                if (argumentIndex is null or < 0)
                 {
                     return false;
                 }
 
-                var constructorOrMethod = argument.Ancestors()
-                    .FirstOrDefault(IsInvocationOrObjectCreation);
+                var constructorOrMethod = argument.Ancestors().FirstOrDefault(IsInvocationOrObjectCreation);
                 var methodSymbol = constructorOrMethod != null
                     ? model.GetSymbolInfo(constructorOrMethod).Symbol as IMethodSymbol
                     : null;
 
-                return methodSymbol != null &&
-                    argumentIndex.Value < methodSymbol.Parameters.Length
-                    && methodSymbol.Parameters[argumentIndex.Value].Name.SplitCamelCaseToWords()
-                        .Any(name => CheckedVariableNames.Contains(name));
+                return methodSymbol != null
+                       && argumentIndex.Value < methodSymbol.Parameters.Length
+                       && methodSymbol.Parameters[argumentIndex.Value]
+                                      .Name
+                                      .SplitCamelCaseToWords()
+                                      .Any(name => CheckedVariableNames.Contains(name));
```

Comment:
I was about to ask for `x =>` and then realized this is also an option. Would it work?
```suggestion
                       && methodSymbol.Parameters[argumentIndex.Value]
                                      .Name
                                      .SplitCamelCaseToWords()
                                      .Any(CheckedVariableNames.Contains);
```